course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -181,9 +181,64 @@ describe('Unit testing PeopleController', function() {
 
     });
     describe('confirmAddPeopleToCourse', function() {
-        it('reflects the right role and number of people');
-        it('initiates the process if confirmed');
-        it('doesn\'t initiate the process if canceled');
+        beforeEach(function() {
+            var ci = {
+                course_instance_id: $routeParams.courseInstanceId,
+                title: 'confirmAddPeopleToCourse test',
+            };
+            courseInstances.instances[ci.course_instance_id] = ci;
+            controller = $controller('PeopleController', {$scope: scope});
+        });
+
+        xit('reflects the right role and number of people', function() {
+
+            var searchTerm ="123456789, \n987654321";
+            //scope.selectedRole.roleName = "Student";
+            var searchTermList = [
+                ['123456789', ['123456789']],
+                ['987654321 ', ['987654321']],
+            ];
+            spyOn(scope, 'getSearchTermList').and.returnValue(searchTermList);
+            scope.confirmAddPeopleToCourse(searchTerm);
+
+            var modalScope = scope.modalInstance.scope();
+            expect(modalScope.numPeople).toEqual(2);
+            expect(modalScope.selectedRoleName).toEqual("Student");
+        });
+
+        it('initiates the process if confirmed', function() {
+            var searchTerm ="123456789, \n987654321";
+            var searchTermList = [
+                ['123456789', ['123456789']],
+                ['987654321 ', ['987654321']],
+            ];
+            spyOn(scope, 'getSearchTermList').and.returnValue(searchTermList);
```

Comment:
I don't think we need to mock this, really. It's a straightforward call that doesn't rely on anything external. In any case, if you wanted to continue to mock it, the `searchTermList` var we'd expect it to return would be:
```
["123456789", "987654321"]
```